coalib/bears/PackageRequirement.py

```diff
@@ -0,0 +1,30 @@
+from coala_decorators.decorators import generate_repr
+
+
+@generate_repr()
+class PackageRequirement:
+
+    def __init__(self, manager: str, package: str, version: str):
+        """
+        :param manager: A string with the name of the manager (pip, npm, etc).
+        :param package: A string with the name of the package to be installed.
+        :param version: A number that contains the version.
+        """
+        self.manager = manager
+        self.package = package
+        self.version = version
+
+    def check(self):
+        """
+        Check if the requirement is satisfied.
+        """
+        raise NotImplementedError
+
+
+class PythonRequirement(PackageRequirement):
+
+    def __init__(self, package, version):
+        PackageRequirement.__init__(self, 'pip', package, version)
+
+    def multiple(self, *args):
```

Comment:
I was looking at the example above calling for `PythonRequirement.multiple` this might not work if `multiple` is not a classmethod - if I am correct. You will most likely get a `TypeError` because you are missing 1 positional argument -- `self`.
Let's say we have this code:
```
class PythonRequirement:
    def multiple(self, *args):
        return args

PythonRequirement.multiple('test', 'hey')
```
you will get a `TypeError` telling you
```
TypeError: unbound method multiple() must be called with PythonRequirement instance as first argument (got str instance instead)
```